baremetal/metal3machine_manager_test.go

```diff
@@ -686,7 +693,7 @@ var _ = Describe("Metal3Machine manager", func() {
 
 						OwnerReferences: []metav1.OwnerReference{
 							{
-								APIVersion: "controlplane.cluster.x-k8s.io/v1beta1",
+								APIVersion: "controlplane.cluster.x-k8s.io/v1beta2",
```

Comment:
How would this impact when I read `apiVersion has been replaced with apiGroup. ` in the migration doc?

Reply:
That is only changed for infrastructureRef, controlPlaneRef, and bootstrap.configRef.
For OwnerReference API Version is still there